<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AllTracksWindow</class>
 <widget class="QWidget" name="AllTracksWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Все треки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="styleSheet">
      <string notr="true">font-size: 18pt; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Все треки</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="searchLayout">
     <item>
      <widget class="QComboBox" name="comboBoxFilter">
       <item>
        <property name="text">
         <string>Все поля</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Название</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Исполнитель</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Жанр</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Год</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Длительность</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditSearch">
       <property name="placeholderText">
        <string>Поиск...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableViewTracks"/>
   </item>
   <item>
 <layout class="QHBoxLayout" name="buttonLayout">
  <item>
   <spacer name="horizontalSpacer">
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
    <property name="sizeType">
     <enum>QSizePolicy::Policy::Expanding</enum>
    </property>
    <property name="sizeHint" stdset="0">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </spacer>
  </item>
  <item>
   <widget class="QPushButton" name="btnAddToPlaylist">
    <property name="text">
     <string>Добавить в плейлист</string>
    </property>
   </widget>
  </item>
 </layout>
</item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
